src/test/scala/nl.knaw.dans.easy.deposit/TestSupportFixture.scala

```diff
@@ -30,12 +30,29 @@ import org.scalamock.scalatest.MockFactory
 import org.scalatest._
 import org.scalatest.enablers.Existence
 
+import scala.util.Properties
+
 trait TestSupportFixture extends FlatSpec with Matchers with Inside with BeforeAndAfterEach with MockFactory {
   implicit def existenceOfFile[FILE <: better.files.File]: Existence[FILE] = _.exists
 
   lazy val testDir: File = currentWorkingDirectory / "target" / "test" / getClass.getSimpleName
   lazy val uuid: UUID = UUID.randomUUID()
 
+  lazy val (user, userGroup, unrelatedGroup) = {
```

Comment:
looks like you copied this from [`easy-split-multi-deposit`](https://github.com/DANS-KNAW/easy-split-multi-deposit/blob/ea7c2dc3d6a78cacd779d2efcb2d799e399e22cf/src/test/scala/nl.knaw.dans.easy.multideposit/actions/SetDepositPermissionsSpec.scala#L28-L41)... Please state so in a comment for future reference!